app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/BottomNavigationBehavior.kt

```diff
@@ -53,7 +53,7 @@ open class BottomNavigationBehavior : CoordinatorLayout.Behavior<BottomNavigatio
     internal fun updateSnackBarPaddingBottomByBottomNavigationView(view: BottomNavigationView) {
         snackbar?.apply {
             val translateY = (view.height - view.translationY).toInt()
-            setPadding(paddingLeft, paddingTop, paddingRight, translateY)
+            setPadding(paddingEnd, paddingTop, paddingStart, translateY)
```

Comment:
👀 I guess here should be `setPaddingWithLayoutDirction(paddingStart, paddingTop, paddingEnd, translateY)`

Reply:
😂